runtime/helpers/elementHelpers.js

```diff
@@ -2,62 +2,75 @@
 
 const waitHelpers = require("./waitHelpers");
 
-async function getSelectOptions(selectSelector) {
-	const options = await driver.$$(selectSelector + ' > option');
-	return Promise.all(
-		options.map(async (opt) => {
-			return {
-				text: (await opt.getHTML(false)).trim(),
-				value: await opt.getAttribute("value"),
-			};
-		})
-	);
-};
+//Wait for element and click (without waitUntilPageLoads)
+async function click(selectorOrElement) {
+    const element = await waitHelpers.waitUntilElementIsClickable(selectorOrElement);
+    await element.click();
+    //waitUntilPageLoads(); is temporary: 
```

Comment:
in the end, if we wait for page load there is one difference between click() and clickAndWait - one more waitUntil page loads
=> do the tests run if we remove the waitUntilPageLoad in function click()?

Reply:
When I finish my current  task https://ticketsystem.hpi-schul-cloud.org/browse/SC-6771: Yes